conference/SE/fse.yml

```diff
@@ -0,0 +1,15 @@
+- title: ESEC/FSE
+  description:  ACM Joint European Software Engineering Conference and Symposium on the Foundations of Software Engineering
+  sub: SE
+  rank: A
+  dblp: sigsoft
+  confs:
+    - year: 2022
+      id: fse22
+      link: https://2022.esec-fse.org/
+      timeline:
+        - paper_registration_deadline: '2022-03-10 23:59:59'
```

Comment:
sorry, there *is not* a field called 'paper_registration_deadline'
consider using 'abstract_deadline' or something like multiple deadlines with different comments.

Reply:
Fixed. Thanks.